### ISSUE TITLE
Semantics for CircularProgressIndicator is missing

### ISSUE THREAD

**Issue by opensourcegps** [flutter/flutter#23369]
When using CircularProgressIndicator as a loading screen, Google Play console pre-launch report complains "This item may not have a label readable by screen readers.".

It's not clear from the documentation how to force Semantics for CircularProgressIndicator or to have some hidden label to pass such accessibility tests.

## Steps to Reproduce

````
  return new Scaffold(
      body: new Center(
        child: new CircularProgressIndicator(),  
      ),

    );
````

## Result

![flutter_loading](https://user-images.githubusercontent.com/7678445/47297635-eafe6d00-d615-11e8-8cf3-75b7a23ab9c5.png)

 

```
[✓] Flutter (Channel beta, v0.9.4, on Linux, locale en_US.UTF-8)
    • Flutter version 0.9.4 at /home/flutter/flutter_v0.9.4-beta
    • Framework revision f37c235c32 (4 weeks ago), 2018-09-25 17:45:40 -0400
    • Engine revision 74625aed32
    • Dart version 2.1.0-dev.5.0.flutter-a2eb050044

[✓] Android toolchain - develop for Android devices (Android SDK 27.0.3)
    • Android SDK at /home/android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-27, build-tools 27.0.3
    • ANDROID_HOME = /home/android/sdk
    • Java binary at: /home/android/src/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)
    • All Android licenses accepted.

[✓] Android Studio (version 3.1)
    • Android Studio at /home/android/src/android-studio
    • Flutter plugin version 24.2.1
    • Dart plugin version 173.4700
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)

[✓] Connected devices (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 8.1.0 (API 27) (emulator)

• No issues found!

```

**Comment by dnfield** [comment]
Duplicate of https://github.com/flutter/flutter/issues/14465

**Comment by dnfield** [comment]
Thanks for the report!  Please follow the other bug for updates, or feel free to reopen/open a new issue if you think this is significantly different.
